cfpb/cfgov-refresh
# Bump ccdb5-api repo to version 1.0.7

This change bumps the version of the optional [ccdb5-api](https://github.com/cfpb/ccdb5-api) repo from 1.0.6 to 1.0.7. This change adds Django 1.11 support and improves dependency compatibility with cfgov-refresh.

## Testing

To properly test this locally against real data, you need to load a local Elasticsearch with complaint data. I was able to do this using the https://github.com/cfpb/ccdb-data-pipeline repository, although it takes a little while to run end-to-end and requires [commenting out one line](https://github.com/cfpb/ccdb-data-pipeline/issues/20) to get running locally.

Assuming you've installed both ccdb5-ui and the new ccdb5-api wheel from this PR, your local Elasticsearch is running, and  your`ES_HOST` and `ES_PORT` environment variables are set:

1. Load data into your ES using this command from ccdb-data-pipeline:

   `python run_pipeline.py -i complaint-index`

   This creates an index named `complaint-index`. This command takes a long time to complete.
2. Run your local cfgov server using these environment variables:

   `COMPLAINT_ES_INDEX=complaint-index COMPLAINT_DOC_TYPE=complaint ./runserver.sh`
3. Visit http://localhost:8000/data-research/consumer-complaints/search/ and you should see results and be able to search them.

## Checklist
